python/lsst/ts/observatory/control/data/atcalsys_schema.yaml

```diff
@@ -0,0 +1,77 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_observatory_control/blob/develop/python/lsst/ts/observatory/control/data/atcalsys_schema.yaml
+title: Configuration file validation schema of Auxiliary Telescope Calibration System
+type: object
+properties:
+  calib_type:
+    type: string
+    enum:
+      - Mono
+      - WhiteLight
+  use_camera:
+    type: boolean
+  atspec_filter:
+    type: string
+    enum:
```

Comment:
These here are not enums. They can change to other things beyond these values, so you can leave it as a free form string.